pkg/sbom/io/decode.go

```diff
@@ -235,7 +235,7 @@ func (m *Decoder) decodePackage(ctx context.Context, c *core.Component) (*ftypes
 			pkg.FilePath = f.Path
 		}
 		// An empty path represents a package digest
-		if f.Path == "" && len(f.Digests) > 0 {
+		if len(f.Digests) > 0 {
```

Comment:
@knqyf263 Can we discuss about this again? :) (cf. https://github.com/aquasecurity/trivy/pull/6310#discussion_r1527746700)